README.md

```diff
@@ -9,6 +9,52 @@ submissions hold. Its evolution is coupled with the [hepdata-pyhf-visualizer][py
 JS package, as this one renders what gets summarized by this component.
 
 
+## Installation
+
+Via pip:
+```sh
+pip install hepdata-pyhf-extractor
+```
+
+Via GitHub (for developers):
+```
+git clone https://github.com/HEPData/hepdata-pyhf-extractor
+cd hepdata-pyhf-extractor
+pip install ".[dev]"
+```
+
+
+## Usage
+
+To extract basic information out of a `pyhf` Pallet file, the summarizers can:
+
+A) Receive a Pallet file path for them to open and read it internally.
+
+```python3
+import gzip
+import json
+from pyhf_extractor import V1PalletSummarizer
+
+pallet_path = "example_pallet.json"
+
```

Comment:
This file `example_pallet.json` is not gzipped, so `gzip.open` will fail.  I guess you can choose either (i) `example_pallet.json.gz` and open with `gzip.open`, or (ii) `example_pallet.json` and open with `open`.

Reply:
Thanks for the catch.

I am unsure how this incompatibility, and the python snippet swap happened... 🤦🏻